Example/Podfile

```diff
@@ -16,7 +16,7 @@ abstract_target 'core' do
   end
 
   target 'ObjectiveCExampleTests_macOS' do
-    platform :osx, '10.8'
+    platform :osx, '10.9'
```

Comment:
Let's change it in the `SSZipArchive.podspec` too: we're not maintaining 10.8 anymore.

Reply:
I'm sure I changed that. Are you looking at the latest Fixed Example Tests commit?

Reply:
Oh, I missed it.